iphone/Maps/Classes/MapViewController.mm

```diff
@@ -526,7 +526,7 @@ - (void)processMyPositionPendingTimeout {
     [object processMyPositionPendingTimeout];
```

Comment:
0. This callback can be called in the background, right?
1. What is done here?
2. The correct implementation should never call `processMyPositionPendingTimeout` when the app is activated, right? Is there a way to implement it properly?